.gitignore

```diff
@@ -131,3 +131,4 @@ local.properties
 .settings
 .classpath
 .factorypath
+/examples/quickstart/nbactions.xml
```

Comment:
netbeans?

Reply:
Yup.

Reply:
I can maybe make it more general (**/nb**.xml) ?

Reply:
Made it more general.